rel/server/env.sh.eex

```diff
@@ -1,39 +1,40 @@
-if [ "$LIVEBOOK_CLUSTER" = "auto" ] && [ ! -z "$FLY_APP_NAME" ]; then
-  export LIVEBOOK_CLUSTER="dns:${FLY_APP_NAME}.internal"
+if [[ "$RELEASE_COMMAND" == "rpc" || "$RELEASE_COMMAND" == "remote" ]]; then
+  export RELEASE_DISTRIBUTION="name"
+  if [ ! -z "${LIVEBOOK_NODE}" ]; then export RELEASE_NODE=${LIVEBOOK_NODE}; fi
+  if [ ! -z "${LIVEBOOK_COOKIE}" ]; then export RELEASE_COOKIE=${LIVEBOOK_COOKIE}; fi
+else
+  if [ "$LIVEBOOK_CLUSTER" = "auto" ] && [ ! -z "$FLY_APP_NAME" ]; then
+    export LIVEBOOK_CLUSTER="dns:${FLY_APP_NAME}.internal"
 
-  case "$ERL_AFLAGS $ERL_ZFLAGS" in
-    *"-proto_dist"*) ;;
-    *)
-      export ERL_AFLAGS="$ERL_AFLAGS -proto_dist inet6_tcp"
-      ;;
-  esac
+    case "$ERL_AFLAGS $ERL_ZFLAGS" in
+      *"-proto_dist"*) ;;
+      *)
+        export ERL_AFLAGS="$ERL_AFLAGS -proto_dist inet6_tcp"
+        ;;
+    esac
 
-  if [ -z "${LIVEBOOK_NODE}" ]; then
-    deployment="$(echo "$FLY_IMAGE_REF" | shasum | cut -c 1-10)"
-    export LIVEBOOK_NODE="${FLY_APP_NAME}-${deployment}@${FLY_PRIVATE_IP}"
+    if [ -z "${LIVEBOOK_NODE}" ]; then
+      deployment="$(echo "$FLY_IMAGE_REF" | shasum | cut -c 1-10)"
+      export LIVEBOOK_NODE="${FLY_APP_NAME}-${deployment}@${FLY_PRIVATE_IP}"
+    fi
   fi
-fi
 
-if [ -f "${RELEASE_ROOT}/user/env.sh" ]; then
-  . "${RELEASE_ROOT}/user/env.sh"
-fi
-
-export RELEASE_MODE="interactive"
-if [ -z "${RELEASE_DISTRIBUTION}" ]; then export RELEASE_DISTRIBUTION="none"; fi
+  if [ -f "${RELEASE_ROOT}/user/env.sh" ]; then
+    . "${RELEASE_ROOT}/user/env.sh"
+  fi
 
-# Mirror these values, so that it is easier to use "bin/release rpc",
-# though it still requires setting RELEASE_DISTRIBUTION=name
-if [ ! -z "${LIVEBOOK_NODE}" ]; then export RELEASE_NODE=${LIVEBOOK_NODE}; fi
-if [ ! -z "${LIVEBOOK_COOKIE}" ]; then export RELEASE_COOKIE=${LIVEBOOK_COOKIE}; fi
```

Comment:
We can remove it here as well:

https://github.com/livebook-dev/livebook/blob/da565fc632ef1f5906d828f11b1a00ae7cf49263/rel/server/env.bat.eex#L8-L9

(note that it would've been used with releases for remote/rpc, but it was just for consistency)

Reply:
We need to keep those since LIVEBOOK_NODE and LIVEBOOK_COOKIE are the variables we expect to be set. We actually don't work if you set RELEASE_COOKIE.

Reply:
I am not following, in our case setting `RELEASE_COOKIE` and `RELEASE_COOKIE` only makes sense for remote/rpc (as you already moved it), so my point was to make the same for windows.